### PR TITLE
CI: Run also on Ubuntu on AppVeyor

### PR DESCRIPTION
**Pull request by 304NotModified**
- [x] `msbuild` to `dotnet`
- [x] target master

### CODE REVIEW

**Comment by 304NotModified** [comment]
now targeting Normalize-Line-Endings branch to prove the tests

**Comment by 304NotModified** [comment]
green combined with Normalize-Line-Endings :)

**Comment by 304NotModified** [comment]
maybe before undrafting, move all to `dotnet`. And maybe build more platforms on Linux

File: appveyor.yml [hunk 1]
```diff
@@ -1,15 +1,45 @@
...
-  - nuget.exe install OpenCover -ExcludeVersion
-  - OpenCover\tools\OpenCover.Console.exe -register:user -target:"C:/Program Files/dotnet/dotnet.exe" -targetargs:"test -f net5.0  -c debug YAXLibTests" -filter:"+[YAXLib]*" -excludebyattribute:*.ExcludeFromCodeCoverage* -hideskipped:All -output:coverage.xml -oldstyle
-  - "SET PATH=C:\\Python34;C:\\Python34\\Scripts;%PATH%"
-  - pip install codecov
-  - codecov -f "coverage.xml"
+for:
+  -
+    matrix:
+      only:
+        - image: Visual Studio 2019
```
Comment (axunonb) [inline appveyor.yml:17]:
```suggestion
        - image: Visual Studio 2019
        - job_name: windows
```

File: appveyor.yml [hunk 2]
```diff
@@ -1,15 +1,45 @@
...
+      - dotnet test YAXLibTests
+      - nuget.exe install OpenCover -ExcludeVersion
+      - OpenCover\tools\OpenCover.Console.exe -register:user -target:"C:/Program Files/dotnet/dotnet.exe" -targetargs:"test -f net5.0  -c debug YAXLibTests" -filter:"+[YAXLib]*" -excludebyattribute:*.ExcludeFromCodeCoverage* -hideskipped:All -output:coverage.xml -oldstyle
+      - "SET PATH=C:\\Python34;C:\\Python34\\Scripts;%PATH%"
+      - pip install codecov
+      - codecov -f "coverage.xml"
+  -
+    matrix:
+      only:
+        - image: Ubuntu
```
Comment (axunonb) [inline appveyor.yml:33]:
```suggestion
        - image: Ubuntu
        - job_name: linux
```

**Comment by 304NotModified** [comment]
I propose:
Merge Normalize-Line-Endings
Then this one (after change target branch)

File: appveyor.yml [hunk 3]
```diff
@@ -1,15 +1,47 @@
-clone_folder: c:\xaxlib
 version: 2.16.{build}
-image: Visual Studio 2019
+image:
+  - Ubuntu
+  - Visual Studio 2019
+
```
Comment (axunonb) [inline appveyor.yml:5]:
```suggestion
environment:
  matrix:
  - job_name: windows
  - job_name: linux
```

**Review by axunonb — COMMENTED** [review]
[Looks like](https://www.appveyor.com/docs/job-workflows/) a job must be defined before its name is used. Not sure...

File: appveyor.yml (hunk 3 above)
Comment (304NotModified) [reply appveyor.yml]: Think you're rights. Bit unclear in the docs.

Comment (304NotModified) [reply appveyor.yml]: AppVeyor don't like it

**Comment by 304NotModified** [comment]
Looks good again 😃

**Comment by axunonb** [comment]
> 
> 
> Looks good again 😃

Thanks, this was the missing piece, and now the badges work.
image

**Comment by 304NotModified** [comment]
I'm not happy with the merge commit here :'( 

I like to rebase, but that's hard now

**Comment by 304NotModified** [comment]
force pushed removed the merge

@axunonb please note, we squash merge the pull requests now, so a merge will lead to more duplicate commits

**Review by axunonb — APPROVED** [review]
Good team work, tnx ;-)

**Pull request merged**